### PR TITLE
Automatically bump Elastic stack version in EC/ECK pipelines

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

EC and ECK pipelines were not part of the automated process to bump the elastic stack releases.
For such:
- I enabled the `7.x` branch , this is an alias and will be transformed to 7.<minor>.<patch>-SNAPSHOT accordingly.
- I used the latest 7.release and delegate that automation to the Elastic stack bump process.

## Why is it important?

Fix the broken changes in the current runs that use 7.12....

## Test

Given

```bash
$ .ci/bump-stack-release-version.sh 7.13.2
Update stack with versions 7.13.2 in scripts/modules/cli.py
No required changes in the scripts/modules/cli.py
Update stack with versions 7.13.2 in tests/versions/apm_server.yml
No required changes in the tests/versions/apm_server.yml
Update stack with versions 7.13.2 in .ci/scripts/common.sh
Update stack with versions 7.13.2 in .ci/integrationTestEC.groovy
Update stack with versions 7.13.2 in .ci/integrationTestECK.groovy
[feature/eck-bump-automation 76bd2da] [Automation] Update elastic stack release version to 7.13.2
 4 files changed, 4 insertions(+), 4 deletions(-)
commit 76bd2dac0c36f4cb1a09219a591de7a94bd33b41 (HEAD -> feature/eck-bump-automation)
Author: Victor Martinez <VictorMartinezRubio@gmail.com>
Date:   Fri Jun 18 11:30:44 2021 +0100

    [Automation] Update elastic stack release version to 7.13.2
You can now push and create a Pull Request
```

produces:

```diff
diff --git a/.ci/integrationTestEC.groovy b/.ci/integrationTestEC.groovy
index 276181b..deb281d 100644
--- a/.ci/integrationTestEC.groovy
+++ b/.ci/integrationTestEC.groovy
@@ -59,7 +59,7 @@ pipeline {
             // The below line is part of the bump release automation
             // if you change anything please modifies the file
             // .ci/bump-stack-release-version.sh
-            values '8.0.0-SNAPSHOT', '7.x', '7.12.0'
+            values '8.0.0-SNAPSHOT', '7.x', '7.13.2'
           }
         }
         stages {
diff --git a/.ci/integrationTestECK.groovy b/.ci/integrationTestECK.groovy
index fd6061f..76d5d40 100644
--- a/.ci/integrationTestECK.groovy
+++ b/.ci/integrationTestECK.groovy
@@ -61,7 +61,7 @@ pipeline {
             // The below line is part of the bump release automation
             // if you change anything please modifies the file
             // .ci/bump-stack-release-version.sh
-            values '7.x', '7.12.0'
+            values '7.x', '7.13.2'
           }
         }
         stages {
diff --git a/.ci/scripts/common.sh b/.ci/scripts/common.sh
index 878de68..7245999 100644
--- a/.ci/scripts/common.sh
+++ b/.ci/scripts/common.sh
@@ -79,7 +79,7 @@ if [ -z "${DISABLE_BUILD_PARALLEL}" ] || [ "${DISABLE_BUILD_PARALLEL}" = "false"
  BUILD_OPTS="${BUILD_OPTS} --build-parallel"
 fi
 
-ELASTIC_STACK_VERSION=${ELASTIC_STACK_VERSION:-'7.13.1'}
+ELASTIC_STACK_VERSION=${ELASTIC_STACK_VERSION:-'7.13.2'}
 
 echo "ELASTIC_STACK_VERSION=${ELASTIC_STACK_VERSION}"
 echo "APM_SERVER_BRANCH_VERSION=${APM_SERVER_BRANCH_VERSION}"
diff --git a/scripts/modules/cli.py b/scripts/modules/cli.py
index 89e31ed..d558998 100644
--- a/scripts/modules/cli.py
+++ b/scripts/modules/cli.py
@@ -80,7 +80,7 @@ class LocalSetup(object):
         '7.10': '7.10.3',
         '7.11': '7.11.2',
         '7.12': '7.12.1',
-        '7.13': '7.13.1',
+        '7.13': '7.13.2',
         'master': '8.0.0',
     }
 

```

### CODE REVIEW

File: .ci/integrationTestEC.groovy [hunk 1]
```diff
@@ -55,14 +55,17 @@ pipeline {
         agent { label 'linux && immutable' }
         axes {
           axis {
-              name 'ELASTIC_STACK_VERSION'
-              values '8.0.0-SNAPSHOT', '7.12.1-SNAPSHOT', '7.12.0'
+            name 'STACK_VERSION'
+            // The below line is part of the bump release automation
+            // if you change anything please modifies the file
+            // .ci/bump-stack-release-version.sh
+            values '7.x', '7.12.0'
```
Comment (v1v) [inline .ci/integrationTestEC.groovy:62]: `7.12.0` should be updated automatically when this PR gets merged

File: .ci/integrationTestECK.groovy [hunk 2]
```diff
@@ -57,14 +57,17 @@ pipeline {
         agent { label 'linux && immutable' }
         axes {
           axis {
-              name 'ELASTIC_STACK_VERSION'
-              values '7.13.0-SNAPSHOT', '7.12.1-SNAPSHOT', '7.12.0'
+            name 'STACK_VERSION'
+            // The below line is part of the bump release automation
+            // if you change anything please modifies the file
+            // .ci/bump-stack-release-version.sh
+            values '7.x', '7.12.0'
```
Comment (v1v) [inline .ci/integrationTestECK.groovy:64]: ditto.

Let's test with the latest 7 snapshot for simplicity

Comment (v1v) [reply .ci/integrationTestECK.groovy:64]: Or do we need to support 8.0-SNAPSHOT too?

Comment (kuisathaverat) [reply .ci/integrationTestECK.groovy:64]: last time I tested 8.0.0-SNAPSHOT did not work with ECK, but probably it do now

**Review by kuisathaverat — APPROVED** [review]

File: .ci/integrationTestECK.groovy (hunk 2 above)
Comment (v1v) [reply .ci/integrationTestECK.groovy]: I'll try it once this is merged in another follow up then

**Review by cachedout — APPROVED** [review]

**Pull request merged**